cypher/cypher-docs/src/docs/dev/ql/administration/security/user-management-syntax.asciidoc

```diff
@@ -1,9 +1,9 @@
 .User management command syntax
-[options="header", width="100%", cols="5a,3,2,^,^."]
+[options="header", width="100%", cols="7a,3,2"]
 |===
-| Command | Description | Required privilege | Community Edition | Enterprise Edition
+| Command | Description | Required privilege
 
-| [source, cypher]
```

Comment:
I think we have these cypher everywhere (and they exist in both the create user and alter user files you updated in this PR as well). Not sure how this will affect the built docs.